asc2018/vdr/electrical/WaterlooMidnightSun_ASC2018VDR_Electrical.tex

```diff
@@ -81,65 +79,54 @@
-\bottomrule
-\end{tabular}
-\end{table}
+Due to the different CAN specifications of the Tritium WaveSculptor motor controllers, they are allocated a separate powertrain CAN network which interfaces with the primary CAN bus via dedicated transceiver boards. For similar reasons, we also allocate separate CAN buses for solar sensing and charger control.
 
-\section{Overview}
-Midnight Sun XII's electrical system consists of high- and low-voltage domains that are electrically isolated for safety. The high-voltage system includes the Sunpower E-series solar cells, Nomura maximum power point trackers, Tritium motor controllers and NGM SCM-150 motors. These are externally-sourced components purchased by the team and interface with the vehicle's custom embedded systems.
+\section{Electrical System Design}
 
-The low-voltage system is comprised of custom circuit-boards serving several functions for monitoring and controlling the vehicle, primarily: driver controls, power distribution, battery management, and external lights. Several smaller PCBs are located throughout the vehicle to support various sensor interfacing and data collection functions. Most boards communicate over a unified CAN bus, with some nodes supporting smaller subsystems over I2C or SPI. The low-voltage power rail is normally provided by the main battery pack via DC-DC converters, but can also be switched to an auxiliary \SI{12}{\volt} Ni-MH battery during startup or fault modes.
+\subsection{5.2.C.1}
 
-Due to the different CAN specifications of the Tritium WaveSculptor motor controllers, they are allocated a separate powertrain CAN network which interfaces with the primary CAN bus via dedicated transceiver boards. A block diagram of all electrical systems is shown in Figure \ref{fig:msxii-electrical-full-block-diagram}. A block diagram of only the vehicle's high-voltage systems is shown in Figure \ref{fig:msxii-electrical-hv-block-diagram}.
+\subsubsection{System Diagram}
 
-\begin{figure}
-\centering
-\includegraphics[width=\textwidth]{figures/msxii-electrical-full-block-diagram}
-\caption{Block diagram of all electrical systems}
-\label{fig:msxii-electrical-full-block-diagram}
+\begin{figure}[H]
+    \centering
+    \includegraphics[width=\textwidth,page=4]{figures/msxii-block-diagrams}
+    \caption{Block diagram of the full electrical system}
+    \label{fig:msxii-electrical-full-block-diagram}
 \end{figure}
 
-\begin{figure}
-\centering
-\includegraphics[width=\textwidth]{figures/msxii-electrical-hv-block-diagram}
-\caption{Block diagram of all high-voltage electrical systems}
-\label{fig:msxii-electrical-hv-block-diagram}
+\subsubsection{System Power Requirements}
+
+Our primary power requirements are from our motors. We have two NGM SCM150s with a total peak power of 15kW and total continuous power rating of 7.5kW. We plan on limiting maximum system current to 110A, giving us plenty of headroom for our fuse.
```

Comment:
We chose `110 A` because `(7.5 kW / 160 V) * 2 = 46.87500 A * 2`, but that's using `4.2 V`, which is the maximum of the 18650s, plus like `10 A` of headroom.

Shouldn't we at least be using the nominal (`3.635 V`), in which case we have `(7.5 kW / 130.86 V) * 2 = 57.3131591 * 2 = 114.626318 A`, which is already above `110 A`?

Reply:
We can increase the current limit, but 110A is safe based on wire ratings and fuse derating. If we believe the wire gauge charts, our wire can support up to 215A. Per regulations, our fuse needs to be rated for less than 75% of our wire's current capacity. That means our fuse needs to be less than 160A, so we need to use a 150A fuse. If we derate by 25%, we get 112A.

Reply:
Followup: We've changed our wire and fuse to 175A with a 130A current limit.